include/engine/guidance/turn_classification.hpp

```diff
@@ -0,0 +1,104 @@
+#ifndef OSRM_GUIDANCE_TURN_CLASSIFICATION_HPP_
```

Comment:
This file calculates a set of turn angles available at an intersection. It is currently unused